### DISCUSSION TITLE
transmission-daemon exits with 141 status code

### DISCUSSION THREAD

**Discussion by solsticedhiver** [transmission/transmission#7054] [General]
on armv7h, on a rpi3b, running archlinux-arm, transmission-daemon 4.0.6 is exiting minutes after being launched.

I don't know what that exit code is: 141.

In the log, I can only see that it is trying to connect to some trackers, and fails for some.
And then it dies.

**Comment by tearfur** [comment]
Maybe you're seeing #7035? Not sure what 141 is on your system.
